scalr/resource_scalr_workspace.go

```diff
@@ -306,11 +312,12 @@ func resourceScalrWorkspaceCreate(ctx context.Context, d *schema.ResourceData, m
 
 	// Create a new options struct.
 	options := scalr.WorkspaceCreateOptions{
-		Name:           scalr.String(name),
-		AutoApply:      scalr.Bool(d.Get("auto_apply").(bool)),
-		ForceLatestRun: scalr.Bool(d.Get("force_latest_run").(bool)),
-		Environment:    &scalr.Environment{ID: environmentID},
-		Hooks:          &scalr.HooksOptions{},
+		Name:                      scalr.String(name),
+		AutoApply:                 scalr.Bool(d.Get("auto_apply").(bool)),
+		ForceLatestRun:            scalr.Bool(d.Get("force_latest_run").(bool)),
+		DeletionProtectionEnabled: scalr.Bool(d.Get("deletion_protection_enabled").(bool)),
```

Comment:
What if `deletion_protection_enabled` is omitted? Will it be set to `true`?

Reply:
Yes.

Reply:
Have you tested this?